themes/base/templates/storyViewer.handlebars

```diff
@@ -6,37 +6,35 @@
         <a class="sjrkc-st-read-other-stories sjrk-st-read-other-stories " href="#">{{localizedMessages.message_translateText}}</a>
     </div>
 </div>
-<div lang="{{localizedMessages.language}}" class="sjrk-st-story-viewer-main-container">
-    <div class="sjrk-st-link-container">
-        <button class="sjrk-st-button sjrk-st-story-listen-to sjrkc-st-story-listen-to" type="button"><span class="sjrk-st-button-text">{{localizedMessages.message_listenToText}}</span></button>
-    </div>
-
-    {{#if dynamicValues.story.title}}
-        <h2 class="sjrkc-st-story-title sjrk-st-story-title">{{dynamicValues.story.title}}</h2>
-    {{/if}}
-    {{#if dynamicValues.story.author}}
-        <p class="sjrkc-st-story-author sjrk-st-story-author">{{localizedMessages.message_authorText}}</p>
-    {{/if}}
-
-    <div hidden class="sjrk-st-link-container">
-        <a class="sjrkc-st-story-read-more sjrk-st-story-read-more" href="#">{{localizedMessages.message_readMoreText}}</a>
-    </div>
+<div lang="{{localizedMessages.language}}" class="sjrkc-st-story-viewer-main-container sjrk-st-story-viewer-main-container">
+    <div class="sjrkc-st-story-details">
+        {{#if dynamicValues.story.title}}
+            <h2 class="sjrkc-st-story-title sjrk-st-story-title">{{dynamicValues.story.title}}</h2>
+        {{/if}}
+        {{#if dynamicValues.story.author}}
+            <p class="sjrkc-st-story-author sjrk-st-story-author">{{localizedMessages.message_authorText}}</p>
+        {{/if}}
 
-    <div hidden class="sjrk-st-language-container">
-        {{localizedMessages.message_languageLabelText}}: <span class="sjrkc-st-story-language sjrk-st-story-language">
-            {{lookup ../localizedMessages.availableLanguages dynamicValues.story.language}}
-        </span>
-    </div>
+        <div hidden class="sjrk-st-link-container">
+            <a class="sjrkc-st-story-read-more sjrk-st-story-read-more" href="#">{{localizedMessages.message_readMoreText}}</a>
```

Comment:
What is this `read-more` container for? It doesn't seem being used anywhere.

Reply:
@cindyli I'm not sure. This was already present in the file. Perhaps @BlueSlug would know.

Reply:
@cindyli @jobara it's one of many bits of markup leftover from one of the original designs for the tool. The intention was to have a link users could click to see other stories similar to the one they are currently viewing. If/when we implement tagging, this would take advantage of that.

I can file a Jira to remove all such pieces, and in doing so preserve them